src/Compilers/CSharp/Portable/FlowAnalysis/NullableWalker.cs

```diff
@@ -4495,8 +4470,8 @@ internal class LocalState : AbstractLocalState
         internal struct LocalState : AbstractLocalState
 #endif
         {
-            // PROTOTYPE(NullableReferenceTypes): Consider storing nullability rather than non-nullability
-            // or perhaps expose as nullability from `this[int]` even if stored differently.
+            // Consider storing nullability rather than non-nullability
```

Comment:
We should open an issue to track, rather than just removing the word prototype. Otherwise we'll forget this.

Reply:
Changing the storage does not need to be fixed so I think it's reasonable to use a comment without an associated bug.

---
In reply to: [217797157](https://github.com/dotnet/roslyn/pull/29875#discussion_r217797157) [](ancestors = 217797157)